File: 파이썬/2839 설탕 배달.py
sugar = 0
n = int(input())

while True:
    if n % 5 == 0 and n >= 0:
        break
    elif n < 0:
        sugar = -1
        n = 0
        break

    n -= 3
    sugar += 1

print(sugar + (n // 5))

File: java/2839 설탕 배달.java
import java.util.*;

public class Main {
	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		
		int sugar = 0;
		int n = scn.nextInt();

		while (true)
		{
			if (n % 5 == 0 && n >= 0)
				break;
			else if (n < 0)
			{
				sugar = -1;
				break;
			}

			n -= 3;
			sugar++;
		}

		System.out.println(sugar + (n / 5));
		
		
		scn.close();

	}

}
